### PR TITLE
Testnet V2

### PR DESCRIPTION
**Pull request by Kevingislason**
Updates docs on how to send Dolphin private payments for testnet V2

### CODE REVIEW

File: docs/Introduction.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
-# Introduction
+# What is Manta?
 
-Welcome! 👋
+Manta is the universal blockchain platform with on-chain privacy. We are building a scalable and composable blockchain platform with privacy guarantee from the first principle: using cryptographic constructions such as zkSNARKs to allow developers to build all kinds of blockchain applications, including DeFi, NFT, GameFi, DAO and many more with end-to-end privacy guarantees and at the same time interoperable, easy to use, high performance and auditable.
```
Comment (Kevingislason) [inline docs/Introduction.md:3]: Maybe a little wordy / technical. What do you think of:

```
Manta is the universal private blockchain platform. We are building a scalable, composable, and private network from first principles. Manta's zero-knowledge cryptography allows developers to build dapps with end-to-end privacy guarantees, which are also interoperable, easy to use, high-performance and auditable. Manta can bring privacy to DeFi, NFTs, GameFi, DAOs, and countless other blockchain applications.
```

File: docs/Introduction.md [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
-
-Manta plans to offer two networks:
+Manta offers two networks:
 
 - [Calamari Network](https://calamari.network/): Manta's canary network. Calamari is running as a Kusama Parachain.
 - [Manta Network](https://manta.network/): Manta's Polkadot parachain. Manta is looking forward to joining the Polkadot ecosystem!
 
-and a testnet called Dolphin which will run independently of Calamari and Manta.
+## For User
+* Try [Dolphin Testnet dApp](https://app.dolphin.manta.network).
```
Comment (Kevingislason) [inline docs/Introduction.md:11]: Should we also link to the Dolphin tutorial docs here?

File: docs/calamari/RunNode.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 7
+id: RunNode
+title: 🔌  Run a Calamari Node
+---
+
+To run a calamari full node, the most easy way is to use `docker`:
```
Comment (Kevingislason) [inline docs/calamari/RunNode.md:7]: nit: most easy -> easiest

File: docs/concepts/PrivatePayment.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Private Payment
+
+Manta is all about bringing privacy to Web 3.  Supporting privacy in fungible token transfer is essential. The first and most powerful primitive we are building is _Private Payment_, more specifcally, *MantaPay*, a _multi-asset decentralized anonymous payment protocol_. 
+
+The fundamental difference between Manta and other public blockchains is that the fungbile token transfer (a.k.a. payment) is shielded by zero-knowledge proofs and thus to be private. We illustrate the difference below:
```
Comment (Kevingislason) [inline docs/concepts/PrivatePayment.md:5]: 'other' makes it sound like manta is public; I don't think the last sentence is necessary; tweaked to be a little more concise
```
Unlike the vast majority of blockchains, fungbile token transfers (i.e. payments) on Manta are shielded by zero-knowledge proofs, and thus private.
```

File: docs/concepts/PrivatePayment.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+![private payment](./resources/private-payment.png)
+
+## Bring Your Own Token Privacy
+
+One difference between Manta and ZCash is that Manta supports BYOT (Bring Your Own Token) privacy. Manta boostraps from Polakdot/Kusama ecosystem and supports privatize tokens in DOT/SAMA ecosystem via XCM (Cross Chain Messaging). Manta could privatize more tokens via bridges in the future.
```
Comment (Kevingislason) [inline docs/concepts/PrivatePayment.md:11]: nit: I like 'can' over 'could' (it sounds like we are more likely to bridge other tokens)

File: docs/concepts/ShieldedAddress.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Shielded Addresses
+
+One essential concept in MantaPay is **shielded addresses**. A shielded address is a reusable public address for your private assets in Manta. It has the following properties:
```
Comment (Kevingislason) [inline docs/concepts/ShieldedAddress.md:3]: I would remove 'public' here; I think it is a little confusing to say that a 'public address' holds private assets without context. I think the explanation below (2.) is enough

File: docs/concepts/ViewingKey.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Viewing Keys
+
+Each shielded address corresponds to a single **viewing key**. Viewing key allows you to disclose your *complete* transaction history of a shielded address without leaking the spending secrets. This is useful in both accounting and regulation compliance.
```
Comment (Kevingislason) [inline docs/concepts/ViewingKey.md:3]: tweaked slightly for clarity:

Each shielded address corresponds to a single **viewing key**. A viewing key allows you to disclose a shielded address's complete spending history without leaking its spending secret. This is useful for both accounting and regulation compliance.

File: docs/guides/DolphinPay.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+MantaPay allows users to privatize public assets to private, transfer private asset, and convert private asset back to public assets (as demonstrated below).
+![overview](./resources/DolphinPay.svg)
+
+Note: Public assets and private assets have different addresses (public addresses vs shielded addresses). There is no relationship between a public address and a shielded address. 
+
+## Setup Testnet Accounts
+
+### Install a Polkadot.JS Browser Extension and Create a Public Address 
+
+Currently, Dolphin Testnet V2 support Polakdot.JS browser extension wallet to manage public assets.
```
Comment (Kevingislason) [inline docs/guides/DolphinPay.md:12]: nit: supports

File: docs/guides/DolphinPay.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+   `3ZUgqc84wUeFzh2ioRh9yRAM7m8CqunGxsMHHLP6sLus76B3YoLHeQ7jggoV8N1grMv6qu9sLC8oNjHFe2CgJy9s`
+
+   (note that shielded addresses and public addresses have a different format, and are not interchageable)
+
+   Click "Private Transfer" to begin the transaction.
+<br/>
+
+   ![private-transfer](./resources/private-transfer.png)
+
+   Signer will prompt you to enter your password and approve the transaction. Once approved, it will build a zero knowledge proof.
```
Comment (Kevingislason) [inline docs/guides/DolphinPay.md:87]: todo for me: this step could use an image

**Review by Kevingislason — COMMENTED** [review]
Basically looks good, just some phrasing suggestions, let me know what you think

**Review by stechu — APPROVED** [review]

**Pull request merged**